documentation/source/refcard.rst

```diff
@@ -0,0 +1,77 @@
+.. _refcard:
+
+**************
+Reference Card
+**************
+..
+   Document "best practice"; leave out detail.
+   Keep format as 1/3rd width of A4/Letter for taping to monitor frames.
+
+   The "| " syntax is a "Line Block", see
+      https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks
+
+
+| Assign: ``dut.mysignal <= 0xFF00``
+| Assign immediately: ``dut.mysignal.setimmediatevalue(0xFF00)``
+| Assign a metavalue: ``dut.mysignal <= BinaryValue("X")``
+| Read: ``val = dut.mysignal.value``
+|       *(Note:* ``mysig = dut.mysignal`` *creates an alias/reference!)*
+| Bit slice: ``mybit = dut.mysignal[0].value``
+|            ``mybits = dut.mysignal[1:0].value``
```

Comment:
Or should this be
```suggestion
| Bit slice: ``mybit = dut.mysignal.value[0]``
|            ``mybits = dut.mysignal.value[1:0]``
```

Reply:
I guess not:
https://github.com/cocotb/cocotb/blob/8ff91fa9e47ec6a49d3a0bb7f947a62f813cc184/cocotb/handle.py#L534-L550